lib/OpenQA/Task/Job/FinalizeResults.pm

```diff
@@ -44,6 +45,18 @@ sub _finalize_results {
         $minion_job->note(failed_modules => \%failed_to_finalize);
         $minion_job->fail("Finalizing results of $count modules failed");
     }
+    return if $openqa_job->state eq CANCELLED;
+    return if $carried_over;
+    my $key = 'job_done_hook_' . $openqa_job->result;
+    if (my $hook = $ENV{'OPENQA_' . uc $key} // $app->config->{hooks}->{lc $key}) {
+        _done_hook_new_issue($openqa_job, $hook);
+    }
+}
+
+sub _done_hook_new_issue {
+    my ($openqa_job, $hook) = @_;
+    my $id = $openqa_job->id;
+    qx{$hook $id};
 }
```

Comment:
I like the new inline comments by codecov

Reply:
this is definitely covered by tests but it seems the coverage data from the minion job is lost even though I tried to apply the fix in t/10-jobs.t . Will check this.